copyright

```diff
@@ -709,15 +709,15 @@ Comment: Taken from https://www.flickr.com/photos/61135621@N03/17755765778 and c
 
 Files:
  images/land/station28*
-Copyright: mwewering
-License: Pixabay License
-Comment: Taken from https://pixabay.com/photos/corridor-tunnel-architecture-escape-2616414/ and cropped.
+Copyright: N/A (CC0 Public Domain)
+License: CC0
+Comment: Taken from https://pxhere.com/en/photo/429477 and cropped.
 
 Files:
  images/land/station15*
```

Comment:
Where do people keep grabbing this image? This looks like the same exact place as space2.
![space2](https://user-images.githubusercontent.com/17688683/188288331-f24d2a4f-186e-4db9-a9fb-ab386d4d84fc.jpg)
I'd prefer if we had something different.

Reply:
whoops my bad